v2/paths/snapctl.yaml

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+post:
+  tags:
+    - AuthenticationRequired
```

Comment:
I am not entirely sure what the tags influence, but generally `/v2/snapctl` only allows open access to snaps through the snap socket `/run/snapd-snap.socket`.